examples/plot_classification_example.py

```diff
@@ -0,0 +1,247 @@
+
+########################################################################
+# Load the iris dataset and add a spurious feature
+# ----------------------------------------------------------------------
+# We load the iris dataset and add a spurious feature that is a linear combination of
+# the petal length and width. The spurious feature allows to illustrate that
+# PermutationImportance is not robust to spurious features, contrarily to LOCO and CPI.
+dataset = load_iris()
+rng = np.random.RandomState(0)
+X, y = dataset.data, dataset.target
+spurious_feat = X[:, 2] + X[:, 3]
+spurious_feat += rng.normal(size=X.shape[0], scale=np.std(spurious_feat) / 2)
+X = np.hstack([X, spurious_feat.reshape(-1, 1)])
+
+dataset.feature_names = dataset.feature_names + ["spurious_feat"]
+
+
+############################################################################
+# Measure variable importance
+# --------------------------------------------------------------------------
+# Since all three methods compute variable importance as a loss difference, they
+# require a K-fold cross-fitting. Computing the importance for each fold is
+# embarassingly parallel, so we use joblib to parallelize the computation.
+def run_one_fold(X, y, model, train_index, test_index, vim_name="CPI", groups=None):
+    model_c = clone(model)
+    model_c.fit(X[train_index], y[train_index])
+    y_pred = model_c.predict(X[test_index])
+
+    if isinstance(model_c, LogisticRegressionCV):
+        method = "predict_proba"
+        loss = log_loss
+        model_name = "LogReg"
+    else:
+        method = "decision_function"
+        loss = hinge_loss
+        model_name = "SVC"
+    # else:
+    #     method = "predict_proba"
+    #     loss = log_loss
+    #     model_name = "RF"
+
+    if vim_name == "CPI":
+        vim = CPI(
+            estimator=model_c,
+            imputation_model_continuous=RidgeCV(alphas=np.logspace(-3, 3, 10)),
+            n_permutations=50,
+            random_state=0,
+            method=method,
+            loss=loss,
+        )
+    elif vim_name == "LOCO":
+        vim = LOCO(
+            estimator=model_c,
+            loss=loss,
+            method=method,
+        )
+    elif vim_name == "PermutationImportance":
+        vim = PermutationImportance(
+            estimator=model_c,
+            n_permutations=50,
+            random_state=0,
+            method=method,
+            loss=loss,
+        )
+
+    vim.fit(X[train_index], y[train_index], groups=groups)
+    importance = vim.importance(X[test_index], y[test_index])["importance"]
+
+    return pd.DataFrame(
+        {
+            "feature": groups.keys(),
+            "importance": importance,
+            "vim": vim_name,
+            "model": model_name,
+            "score": balanced_accuracy_score(y_true=y[test_index], y_pred=y_pred),
+        }
+    )
+
+
+models = [
+    LogisticRegressionCV(Cs=np.logspace(-3, 3, 10)),
+    GridSearchCV(LinearSVC(), {"C": np.logspace(-3, 3, 10)}),
```

Comment:
These models are very similar. I don't see the benefit of comparing them.

Reply:
I understand. My idea was to show how to change the prediction method (predict_proba & decision_function) and the losses (hinge and log_loss). I suggest changing to an SVC with rbf kernel. WDYT?

Reply:
Sounds reasonable